mne/viz/evoked.py

```diff
@@ -1024,6 +1024,9 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     if picks is not None:
         pick_names = [evoked.info['ch_names'][pick] for pick in picks]
         evoked.pick_channels(pick_names)
+    else:
+        evoked.pick_types(eeg=True, eog=True, seeg=True)
```

Comment:
I would do:
```
if has_eeg:
pick_types(eeg=True, eog=False, seeg=False)
elif has seeg:
pick_types(eeg=False, eog=False, seeg=True)
```
etc

since the default behavior shouldn't plot different chan types at once.